test-ts/utils.ts

```diff
@@ -209,9 +214,9 @@ export async function electGroup(groupAddress: string, voter: SignerWithAddress)
 }
 
 export async function updateGroupSlashingMultiplier(
-  registryContract: MockRegistry,
-  lockedGoldContract: MockLockedGold,
-  validatorsContract: MockValidators,
+  registryContract: any,
```

Comment:
Include comment on why typing was removed

Reply:
I had issues with build locally but it automagically works now - reverted